backend/functions/case-history/case-history.function.test.ts

```diff
@@ -3,49 +3,55 @@ import { NORMAL_CASE_ID, NOT_FOUND_ERROR_CASE_ID } from '../lib/testing/testing-
 import httpTrigger from './case-history.function';
 import { MockHumbleQuery } from '../lib/testing/mock.cosmos-client-humble';
 import { NotFoundError } from '../lib/common-errors/not-found-error';
-
-const context = require('azure-function-context-mock');
+import { createMockAzureFunctionRequest } from '../azure/functions';
 
 describe('Case docket function', () => {
+  const request = createMockAzureFunctionRequest({
+    params: {
+      caseId: '',
+    },
+    method: 'GET',
+  });
+
+  const context = require('azure-function-context-mock');
+
   test('Should return case history for an existing case ID', async () => {
     jest
       .spyOn(MockHumbleQuery.prototype, 'fetchAll')
       .mockResolvedValue({ resources: CASE_HISTORY });
 
     const caseId = NORMAL_CASE_ID;
-    const request = {
+    const requestOverride = {
+      ...request,
       params: {
-        caseId,
+        caseId: caseId,
```

Comment:
We should use the shorthand here:
```suggestion
        caseId,
```